src/main/webapp/js/components/entities/entity-edit.jsx

```diff
@@ -44,7 +48,11 @@ class EntityEdit extends Component {
     });
   };
 
-  validate = values => validate(this.props.fieldConfig)(values);
+  validate = values => validate(
+    this.props.fieldConfig,
+    this.props.entity,
+    this.props.entityToEdit,
```

Comment:
I think it's not exactly the same. Under 'values' there are values provided during entity edit (new ones) and under 'entityToEdit' there are these before edition. They are needed in case of entity edit but without any changes to unique fields. Without this check the changes to entity can't be saved, because this value already exists in the entities list (but it is actually the same entity). I have changed the name of this const to prevValue in form-utils.jsx to be more precise.

Reply:
you can check if the entity from the list has different id than values id to be sure it's different entity, you will not need the entityToEdit then